Libraries/LibJS/Runtime/Object.h

```diff
@@ -39,6 +39,19 @@
 
 namespace JS {
 
+struct PropertyDescriptor {
+    PropertyAttributes attributes;
+    Value value;
+    Function* getter;
+    Function* setter;
```

Comment:
Let's be good bois and initialize these explicitly:
```suggestion
    Function* getter { nullptr };
    Function* setter { nullptr };
```